lib/service/service.go

```diff
@@ -3382,6 +3383,9 @@ func (process *TeleportProcess) initProxyEndpoint(conn *Connector) error {
 				StaticFS:         fs,
 				ClusterFeatures:  process.getClusterFeatures(),
 				ProxySettings:    proxySettings,
+				Installer: installers.InstallerTemplate{
+					AuthServer: process.proxyPublicAddr().Addr,
```

Comment:
I think it's fine to default to connect through proxy since we want this to work out of the box with Cloud. We can just document how to connect directly to auth for whoever wants it.